{
    "assignee": null,
    "body": "I am doing contacts sync using:\r\n$wa->sendSync($contacts, $mode = \"full\", $context = \"registration\", $index = 0, $last = true);\r\n\r\nsometimes I get a Backoff error with the time I should try in e.g. 60 minutes.. How can I intercept this backoff time?\r\n\r\nThanks in advanced..",
    "closed_at": null,
    "comment_data": [
        {
            "body": "listen to `onGetError`",
            "created_at": "2015-01-04T09:18:18Z",
            "html_url": "https://github.com/venomous0x/WhatsAPI/issues/1068#issuecomment-68626425",
            "id": 68626425,
            "issue_url": "https://api.github.com/repos/venomous0x/WhatsAPI/issues/1068",
            "updated_at": "2015-01-04T09:18:18Z",
            "url": "https://api.github.com/repos/venomous0x/WhatsAPI/issues/comments/68626425",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/2076983?v=3",
                "events_url": "https://api.github.com/users/karam89/events{/privacy}",
                "followers_url": "https://api.github.com/users/karam89/followers",
                "following_url": "https://api.github.com/users/karam89/following{/other_user}",
                "gists_url": "https://api.github.com/users/karam89/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/karam89",
                "id": 2076983,
                "login": "karam89",
                "organizations_url": "https://api.github.com/users/karam89/orgs",
                "received_events_url": "https://api.github.com/users/karam89/received_events",
                "repos_url": "https://api.github.com/users/karam89/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/karam89/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/karam89/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/karam89"
            }
        }
    ],
    "comments": 1,
    "comments_url": "https://api.github.com/repos/venomous0x/WhatsAPI/issues/1068/comments",
    "created_at": "2014-12-28T13:24:37Z",
    "event_data": [],
    "events_url": "https://api.github.com/repos/venomous0x/WhatsAPI/issues/1068/events",
    "html_url": "https://github.com/venomous0x/WhatsAPI/issues/1068",
    "id": 52981039,
    "labels": [],
    "labels_url": "https://api.github.com/repos/venomous0x/WhatsAPI/issues/1068/labels{/name}",
    "locked": false,
    "milestone": null,
    "number": 1068,
    "state": "open",
    "title": "How can I know when I get backoff error from Contacts Sync? is there any exception thrown?",
    "updated_at": "2015-01-04T09:18:18Z",
    "url": "https://api.github.com/repos/venomous0x/WhatsAPI/issues/1068",
    "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/10117418?v=3",
        "events_url": "https://api.github.com/users/boxim/events{/privacy}",
        "followers_url": "https://api.github.com/users/boxim/followers",
        "following_url": "https://api.github.com/users/boxim/following{/other_user}",
        "gists_url": "https://api.github.com/users/boxim/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/boxim",
        "id": 10117418,
        "login": "boxim",
        "organizations_url": "https://api.github.com/users/boxim/orgs",
        "received_events_url": "https://api.github.com/users/boxim/received_events",
        "repos_url": "https://api.github.com/users/boxim/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/boxim/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/boxim/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/boxim"
    }
}